core/SConscript.firmware

```diff
@@ -794,6 +794,12 @@ def cargo_build():
         # see https://nnethercote.github.io/perf-book/type-sizes.html#measuring-type-sizes for more details
         env.Append(ENV={'RUSTFLAGS': '-Z print-type-sizes'})
 
+    if ARGUMENTS.get('RUST_EMIT_STACK_SIZES', '0') == '1':
```

Comment:
Since it appears cheap I wonder if we should always include the flag? One less build option to be aware of. Downside is it won't work on stable compiler but I think we're currently far from that anyway.

Reply:
Sounds good - fixed in b508b8a47